### PR TITLE
Added blog post 

### PR DESCRIPTION
**Pull request by vojtechruz**
Added my blog post about gatsbyjs migration to the gatsby blog after request by @KyleAMathews.

Also, wanted for cross posted content on gatsby blog to contain canonical link to the original posts to tell search engines where is the original article. Added Helmet link to the blog post template. However, it looks like gatsby blog uses gatsby-plugin-canonical-urls, which  overrides canonical links back to the gatsby site. 

I am not familar with gatsby plugin development, could someone please add a simple option to exclude certain urls in plugin settings or to work only if canonical link is not already provided? Or somthing similar?

Thanks a lot :)

### CODE REVIEW

File: www/src/templates/template-blog-post.js [hunk 1]
```diff
@@ -49,6 +49,11 @@ class BlogPostTemplate extends React.Component {
         {children}
       </p>
     )
+    let canonicalLink
+    if(post.frontmatter.canonicalLink) {
+      canonicalLink = <link rel="canonical" href={post.frontmatter.canonicalLink} />
```
Comment (vojtechruz) [inline www/src/templates/template-blog-post.js]: The canonical link is overwritten by gatsby-plugin-canonical-urls.

**Comment by KyleAMathews** [comment]
Made some minor edits — will work the canonical url bug and then merge this later. Thanks!

**Comment by KyleAMathews** [comment]
Thanks!

**Comment by KyleAMathews** [comment]
Oh, and the solution to the canonical links problem was pretty easy — just remove gatsby-plugin-canonical-url and use react-helmet for everything — far simpler as they already provide an easy way to set a default value and override it somewhere down the tree.

**Comment by vojtechruz** [comment]
@KyleAMathews Thank you for finishing the PR, appreciate it.

**Comment by jlengstorf** [comment]
Hiya @vojtechruz! 👋

This is definitely late, but on behalf of the entire Gatsby community, I wanted to say thank you for being here.

Gatsby is built by awesome people like you. Let us say “thanks” in two ways:

1.  **We’d like to send you some Gatsby swag.** As a token of our appreciation, you can go to the [Gatsby Swag Store][store] and log in with your GitHub account to get a coupon code good for one free piece of swag. (We’ve got t-shirts and hats, plus some socks that are really razzing our berries right now.)
2.  **If you’re not already part of it, we just invited you to join the Gatsby organization on GitHub.** This will add you to our team of maintainers. You’ll receive an email shortly asking you to confirm. By joining the team, you’ll be able to label issues, review pull requests, and merge approved pull requests.

If you have questions, please don’t hesitate to reach out to us: tweet at [@gatsbyjs][twitter] and we’ll come a-runnin’.

Thanks again! 💪💜

[store]: https://store.gatsbyjs.org
[twitter]: https://twitter.com/gatsbyjs

**Pull request merged**